### PR TITLE
Add section on delegation.

### PR DESCRIPTION
**Pull request by msporny**
This PR is an attempt to address issue #252 by adding a section on delegation proposed by @agropper.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/vc-api/pull/359.html" title="Last updated on Jan 23, 2024, 7:18 PM UTC (098da50)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/vc-api/359/4ce8393...098da50.html" title="Last updated on Jan 23, 2024, 7:18 PM UTC (098da50)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1011,6 +1011,47 @@ <h4>Exchange Examples</h4>
...
+human rights consequences.
+      </p>
+
+      <p>
+<a>Verifiable credentials</a> issuance has a request phase and a delivery phase.
+The request might be made by the <a>subject</a> or another role, and delivery
+can be to a client that might or might not be controlled by the subject.
+Delegation is highly relevant for both phases. The <a>issuer</a> might delegate
+processing of the request to a separate entity. The subject, for their part,
+might also delegate the ability to request a <a>verifiable credential</a> to a
```
Comment (David-Chadwick) [inline index.html:1042]: Note that the subject may not always have the capability or ability to perform delegation. Examples include: a new born baby, a pet, and a person with dementia. So the request might be performed by a third party who was not delegated by the subject.

Comment (msporny) [reply index.html:1042]: Yes, excellent point, I'll work that into the text in the next revision.

**Review by brianorwhatever — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -1011,6 +1011,47 @@ <h4>Exchange Examples</h4>
...
+separate entity. The ability to delegate is a third dimension in the enhanced
+efficiency of processing <a>verifiable credentials</a> and has impact on privacy
+and human rights.
+      </p>
+
+      <p>
+The architecture described in this specification is designed for market acceptance through a combination of
+efficiency and respect for privacy and human rights. APIs and protocols for
+<a>verifiable credentials</a> processing do not favor delegation by the issuer
```
Comment (TallTed) [inline index.html:1051]:
```suggestion
processing <a>verifiable credentials</a> do not favor delegation by the issuer
```

**Review by TallTed — CHANGES_REQUESTED** [review]
A tiny word shuffle. Approval will come after @David-Chadwick's note about externally-imposed delegation is acted upon.

**Review by dlongley — APPROVED** [review]
+1 with David and Ted's suggestions.

File: index.html (hunk 1 above)
Comment (msporny) [reply index.html:1052]: Done in 098da503d35150d8448a05939b301e33202af904.

**Pull request merged**